src/UI/Components/Diagram/actions.ts

```diff
@@ -258,7 +261,7 @@ export function appendInstance(
           if (neighborRelations) {
             const connectedInstance = Array.from(
               neighborRelations,
-              ([id, attrName]) => ({ id, attrName }),
+              ([id, attributeName]) => ({ id, attributeName }),
```

Comment:
Isn't this the same as in line 238? if yes, maybe worth having it as a variable/function